README.md

```diff
@@ -1,4 +1,4 @@
-![PROJECT-0-matic](https://user-images.githubusercontent.com/80773310/194979405-9df5df2d-d368-41bf-8ca8-e4aa040a6fc1.gif)
+https://user-images.githubusercontent.com/80773310/194979405-9df5df2d-d368-41bf-8ca8-e4aa040a6fc1.gif
```

Comment:
Is this working on your side?

Reply:
Looks like it was previously working and now it doesn't. Maybe revert it and I'll double check again.

Reply:
Should I put it like ![](https://user-images.githubusercontent.com/80773310/194979405-9df5df2d-d368-41bf-8ca8-e4aa040a6fc1.gif) this?

Reply:
Either should work I believe. 

Try what you mentioned